src/dctl-transaction-create.js

```diff
@@ -17,11 +19,22 @@ program
     let [transactionType, payload] = program.args;
     if (!transactionType) throw new Error('Error: Missing Param "transactionType"');
     if (!payload) throw new Error('Error: Missing Param "payload"');
-    try {
-      payload = JSON.parse(payload);
-    } catch (e) {
-      console.warn('Could not parse JSON for payload, sending raw data instead...');
+
+    if (program.file) {
+      const payloadSplit = payload.split('.');
+      if (payloadSplit[payloadSplit.length - 1] === 'json') {
```

Comment:
one last nitpick: Files can have json contents without having the extension of `.json`, so we probably shouldn't require.

Instead, just always try to json parse the file contents, and if it fails, then throw the error that the payload must be valid json

Reply:
Implemented in latest commit